### PR TITLE
Categories - move en: translation to top where it exists. For #4082

### PR DESCRIPTION
**Pull request by svensven**

### CODE REVIEW

**Review by hangy — APPROVED** [review]

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -22850,8 +22850,8 @@ fr:Pains de tradition française, pain de tradition française, pains traditionn
 
 <en:Baguettes
 <fr:Pains de tradition française
-fr:Baguettes de tradition française, Baguette de tradition française, baguettes traditionnelles françaises, baguette traditionnelle française, baguettes traditionnelles de France, baguette traditionnelle de France, baguettes tradition, baguette tradition, Baguettes de pain de tradition française
 en:Unsalted French bread, Unsalted baguette
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:22853]: the English name is incorrect, it should be:


```suggestion
en:Traditional French baguette, French traditional baguette
```

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -24405,8 +24405,8 @@ ciqual_food_code:en:23585
 ciqual_food_name:en:Chocolate cake
 
 <en:Chocolate cakes
-fr:Moelleux au chocolat
 en:Chocolate cake with melting centre refrigerated
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:24408]:
```suggestion
en:Chocolate cake with melting centre
```

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -29155,9 +29155,9 @@ ciqual_food_name:en:Pont l'Evêque cheese, from cow's milk
 <en:Cow cheeses
 <en:Uncooked pressed cheeses
 <en:French cheeses
-fr:Reblochons, Reblochon, Reblochons de Savoie, Reblochon de Savoie
 en:Reblochon cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29158]:
```suggestion
en:Reblochon, Reblochon cheeses, Reblochon cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -29504,16 +29504,16 @@ country:en:France
...
 en:Beaufort
+fr:Beaufort
 agribalyse_food_code:en:12105
 ciqual_food_code:en:12105
 ciqual_food_name:en:Beaufort cheese, from cow's milk
 country:en:France
 
 <en:French cheeses
-fr:Bleu des Causses
 en:Causses blue cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29515]:
```suggestion
en:Causses blue cheeses, Causses blue cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -29525,17 +29525,17 @@ country:en:France
 
 <en:French cheeses
 <en:Bries
-fr:Brie de Meaux, Bries de Meaux
 en:Brie de Meaux cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29528]:
```suggestion
en:Brie de Meaux cheeses, Brie de Meaux cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -29525,17 +29525,17 @@ country:en:France
...
+fr:Brie de Meaux, Bries de Meaux
 country:en:France
 agribalyse_food_code:en:12021
 ciqual_food_code:en:12021
 ciqual_food_name:en:Brie de Meaux cheese, from cow's milk
 
 <en:French cheeses
 <en:Bries
-fr:Brie de Melun, Bries de Melun
 en:Brie de Melun cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29537]:
```suggestion
en:Brie de Melun, Brie de Melun cheeses, Brie de Melun cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -29591,8 +29591,8 @@ ciqual_food_name:en:Chabichou cheese, from goat's milk
 <en:French cheeses
 <en:Soft cheeses with bloomy rind
 <en:Cow cheeses
-fr:Chaource
 en:Chaource cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29594]:
```suggestion
en:Chaource, Chaource cheeses, Chaource cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 8]
```diff
@@ -29673,8 +29673,8 @@ ciqual_food_name:fr:Gruyère IGP France
 
 <en:French cheeses
 <en:Soft cheeses with washed rind
-fr:Langres
 en:Langres cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29676]:
```suggestion
en:Langres, Langres cheeses, Langres cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 9]
```diff
@@ -29689,8 +29689,8 @@ country:en:France
 <en:French cheeses
 <en:Uncooked pressed cheeses
 <en:Cow cheeses
-fr:Morbiers
 en:Morbier cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29692]:
```suggestion
en:Morbier, Morbier cheeses, Morbier cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 10]
```diff
@@ -29804,8 +29804,8 @@ ciqual_food_name:en:Cheese, from goat's milk
 <en:Cow cheeses
 <en:Soft cheeses with bloomy rind
 <en:French cheeses
-fr:Saint-Marcellin
 en:Saint-Marcellin cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29807]:
```suggestion
en:Saint-Marcellin, Saint-Marcellin cheeses, Saint-Marcellin cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 11]
```diff
@@ -29839,9 +29839,9 @@ ciqual_food_name:en:Tomme cheese (PDO) from the French Bauges mountains
 <en:Soft cheeses with washed rind
 <en:French cheeses
 <en:Unpasteurised cheeses
-fr:Mont d'Or, vacherin du Haut-Doubs, Vacherin-Mont d'Or
 en:Vacherin cheese or Mont d'or cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29842]:
```suggestion
en:Mont d'Or, Mont d'Or cheeses, Vacherin cheese or Mont d'or cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 12]
```diff
@@ -29853,8 +29853,8 @@ fr:Fromages du Nord-Pas-de-Calais
 <en:Cow cheeses
 <en:French cheeses
 <en:Soft cheeses with washed rind
-fr:Livarot, Livarots
 en:Livarot cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29856]:
```suggestion
en:Livarot, Livarot cheeses, Livarot cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 13]
```diff
@@ -29954,9 +29954,9 @@ nl:Witschimmelkazen
 wikidata:en:Q1256296
 
 <en:Soft cheeses with bloomy rind
-fr:Carrés de l'Est, Carré de l'Est
 en:Carré de l'Est cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:29957]:
```suggestion
en:Carré de l'Est, Carré de l'Est cheeses, Carré de l'Est cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 14]
```diff
@@ -30115,9 +30115,9 @@ ciqual_food_name:en:Edam cheese, from cow's milk
 <en:Cow cheeses
 <en:Uncooked pressed cheeses
 <en:French cheeses
-fr:Saint-nectaire, Saint-nectaires, Saints-nectaires
 en:Saint-Nectaire cheese
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:30118]:
```suggestion
en:Saint-Nectaire, Saint-Nectaire cheeses
```

File: taxonomies/categories.txt [hunk 15]
```diff
@@ -31643,9 +31643,9 @@ ciqual_food_code:en:12030
 ciqual_food_name:en:Maroilles fermier cheese
 
 <en:Cheeses
-fr:Raclettes
 en:Raclette cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:31646]:
```suggestion
en:Raclette, Raclette cheeses, Raclette cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 16]
```diff
@@ -32143,8 +32143,8 @@ origins:en: en:France
 
 <en:French cheeses
 <en:Cooked pressed cheeses
-fr:Comté
 en:Comté cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:32146]:
```suggestion
en:Comté, Comté cheeses, Comté cheese from cow's milk
```

File: taxonomies/categories.txt [hunk 17]
```diff
@@ -33495,8 +33495,8 @@ hu:Kenhető sajtok
 nl:Smeerkazen
 
 <en:Cheeses
-fr:Fromages Ail et fines herbes, fromage ail et fines herbes type Boursin, Tartare
 en:Uncured soft cheese spreadable around 30-40% fat flavoured (ex : garlic and herbs)
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:33498]:
```suggestion
en:Uncured soft cheese spreadable around 30-40% fat flavoured
```

File: taxonomies/categories.txt [hunk 18]
```diff
@@ -33495,8 +33495,8 @@ hu:Kenhető sajtok
...
+fr:Fromages Ail et fines herbes, fromage ail et fines herbes type Boursin, Tartare
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:33499]:
```suggestion
fr:Fromages Ail et fines herbes
```

File: taxonomies/categories.txt [hunk 19]
```diff
@@ -35441,8 +35441,8 @@ nl:Gezouten amandelen
 
 <en:Salted almonds
 <en:Grilled almonds
-fr:Amandes grillées et salées, Amande grillée salée
 en:Almond grilled salted
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:35444]:
```suggestion
en:Salted and grilled almonds, Grilled and salted almonds, Almond grilled salted
```

File: taxonomies/categories.txt [hunk 20]
```diff
@@ -48056,8 +48056,8 @@ ciqual_food_code:en:25058
 ciqual_food_name:en:Duck with sauce (green pepper sauce, hunter-style sauce,etc.)
 
 <en:Duck dishes
-fr:Parmentiers de canard, Parmentier de canard
 en:Shepherd's pie with duck cooked
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:48059]:
```suggestion
en:Shepherd's pie with duck
```

File: taxonomies/categories.txt [hunk 21]
```diff
@@ -49759,8 +49759,8 @@ es:Frijoles refritos
 fr:Haricots refrits
 
 <en:Prepared vegetables
-fr:Ratatouilles, ratatouille, Ratatouille cuisinée
 en:Cooked ratatouille
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:49762]:
```suggestion
en:Ratatouille
```

File: taxonomies/categories.txt [hunk 22]
```diff
@@ -52266,8 +52266,8 @@ ciqual_food_code:en:28924
 ciqual_food_name:en:Cooked pork shoulder, standard, rind less and fatless
 
 <en:Pork
-fr:Rôtis de porc, rôti de porc
 en:Pork roast raw
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:52269]:
```suggestion
en:Pork roasts
```

**Review by stephanegigandet — CHANGES_REQUESTED** [review]
A lot of the entries where French was first came from the French CIQUAL database, and the English translations were not good (too specific, or with commas etc.). I made suggestions to change them.

**Comment by svensven** [comment]
Github's not letting me apply the suggestions, either individually or as a batch. it just says "This diff has recently been updated. Refresh and try again". Even tried in a different browser. :facepalm: 
Looks like my browser's trying to POST to ```https://github.com/openfoodfacts/openfoodfacts-server/pull/4098/apply_suggestions``` and getting a 404 back.

**Comment by stephanegigandet** [comment]
> Github's not letting me apply the suggestions, either individually or as a batch. it just says "This diff has recently been updated. Refresh and try again". Even tried in a different browser.

I have this often on the taxonomies for some reason. :( Sorry for all the manual work.

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**